index.Rmd

```diff
@@ -3,19 +3,26 @@ title: "R Development Guide"
 author: "R Contribution Working Group"
 date: "`r Sys.Date()`"
 site: bookdown::bookdown_site
+output: bookdown::gitbook
 documentclass: book
 bibliography: [book.bib]
 biblio-style: apalike
 link-citations: yes
 github-repo: forwards/rdevguide
 url: 'TDDO'
+cover-image: "img/Rlogo.png"
```

Comment:
I think you need to create an image of a book cover using the R logo as the main image. See for example the images on https://bookdown.org/ - they are the cover image of the hard copy book. We don't have a hard copy book of course, but you could create a cover page as a PDF to use as an image (e.g. with PDF export from Word or some other program) . Actually the image for "R Programming for Data Science" looks like a self-made example. So you just need the title, the image and R Contribution Working Group as the author.

Consider how it will look on Twitter, e.g. https://twitter.com/straightedge/status/1203515396910772230?s=20&t=3ApdM_TKev1KPzMP3IW7WA - either make the whole image square or center the image vertically on a larger page so that only the R logo shows on the Twitter image.